.github/workflows/tests.yml

```diff
@@ -7,19 +7,24 @@ on:
       - master
     tags:
       - '*'
+    paths-ignore:
+      - '**/*.md'
+      - 'LICENSE'
+      - 'Dockerfile'
+      - 'docker-compose.yml'
+      - 'dc.build.node.yml'
   pull_request:
     branches:
       - develop
       - master
 
 jobs:
-
   check_skip:
     runs-on: ubuntu-latest
     if: "! startsWith(github.event.head_commit.message, '[CI Skip]') && github.repository == 'kiltprotocol/sdk-js'"
     steps:
-    - name: Not Skipped
-      run: echo "Not Skipped"
+      - name: Not Skipped
+        run: echo "Not Skipped"
 
```

Comment:
@tjwelde can this go? this was just a test right?

Reply:
It is necessary, when we don't want to repeat the `if` line in it in every job

Reply:
👍